File: app/src/main/java/bj/discogsbrowser/model/release/Track.java

package bj.discogsbrowser.model.release;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Track
{

    @SerializedName("duration")
    @Expose
    private String duration;
    @SerializedName("position")
    @Expose
    private String position;
    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("type_")
    @Expose
    private String type;

    public String getDuration() {
        return duration;
    }

    public void setDuration(String duration) {
        this.duration = duration;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

}

File: app/src/main/java/bj/vinylbrowser/model/release/Track.kt
package bj.vinylbrowser.model.release

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class Track(val duration: String,
                 val position: String,
                 val title: String,
                 @SerializedName("type_") val type: String) : Parcelable {

    override fun writeToParcel(dest: Parcel, flags: Int) {
        dest.writeString(duration)
        dest.writeString(position)
        dest.writeString(title)
        dest.writeString(type)
    }

    override fun describeContents(): Int {
        return 0
    }
}